packages/Algernon.package/ALGCore.class/instance/checkWorldBounds.st

```diff
@@ -0,0 +1,6 @@
+visibility
+checkWorldBounds
```

Comment:
This is an Edgecase, but we might still want to check if our bounds are smaller than the Worldbounds meaning negative; E.g. Algernon was moved out of view to the top/left instead of the bottom/right.

Reply:
I don't agree. I don't see why you should do that, but that can only happen if the user deliberately does it. We should not overrule the user.